lib/presentation/pages/tip_information_page.dart

```diff
@@ -89,9 +89,24 @@ class TipInformationPage extends StatelessWidget {
                                       IconButton(
                                         onPressed: () {
                                           if (link == null) {
+                                            ScaffoldMessenger.of(context)
+                                                .showSnackBar(
+                                              const SnackBar(
+                                                content: Text(
+                                                  'Link is not available',
+                                                ),
+                                              ),
+                                            );
```

Comment:
Replace with SnackBarService

Reply:
I apologise, I was using resolutions to keep track of what I solved

Reply:
My tip is to write "done" or whatever is needed on the comments and click on "Submit review" just before I push my updated code.

In the "Files changed" tab before you click "Submit review" I will not see the changes unless you click on "Add single comment" 

If you want to do the same in "Conversation" tab I think it will work just after you start review in the "Files changed"

Hope that will help